include/nbl/builtin/hlsl/shapes/beziers.hlsl

```diff
@@ -507,9 +512,14 @@ struct Quadratic
 };
 
 // This function returns the analytic quartic equation to solve for lhs bezier's t value for intersection with another bezier curve
-template<typename float_t>
+template<typename float_t, typename device_caps = void>
```

Comment:
don't take the device_caps, you should be able to "detect" what `float_t` you have with `is_same_v`